src/components/Proposal/ProposalComments.css

```diff
@@ -35,3 +35,12 @@
   border: 1px solid rgba(115,110,125,.32);
   width: 100%;
 }
+
+@media only screen and (max-width: 425px) {
```

Comment:
Why `425px`?  I'd change this to min-width with a mobile first approach.
(We should talk to @heyimjuani about what breakpoints are we supporting.)

Reply:
Don't have this info off the top of my head but I'd be a +1 for going through the entire platform to review and normalize.

Reply:
Hmm I'm not sure what approach is best.

I'm following what we did with @heyimjuani for the `Finish at` [label on the proposal list](https://github.com/decentraland/governance/blob/4951c4f8d08775bc89e9150ba2df956d97e1ddb8/src/components/Proposal/ProposalItem.css#L16).
The `425px` in this case is because from that screen size onwards, the font size doesn't need a fix to fit.

Do you think maybe we can use this pattern until we meet and review everything?